exercises/concept/gross-store/.docs/instructions.md

```diff
@@ -41,7 +41,7 @@ fmt.Println(bill)
 To implement this, you'll need to:
 
 - Return `false` if the given `unit` is not in the `units` map.
-- Otherwise add the item to the customer `bill`, indexed by the item name, then return `true`.
+- Otherwise add the item to the customer `bill`, indexed by the item name, then return `true`. If the item is already present in the bill, increase its quantity by `unit` amount
```

Comment:
```suggestion
- Otherwise add the item to the customer `bill`, indexed by the item name, then return `true`.
- If the item is already present in the bill, increase its quantity by the amount that belongs to the provided `unit`.
```

Reply:
Yes that looks better.